Lib/test/test_c_locale_coercion.py

```diff
@@ -181,6 +183,22 @@ def setUpModule():
         CLI_COERCION_TARGET = AVAILABLE_TARGETS[0]
         CLI_COERCION_WARNING = CLI_COERCION_WARNING_FMT.format(CLI_COERCION_TARGET)
 
+    # Determine the platform's default LC_CTYPE from the default locale.
+    # POSIX provides that there *exists* a default locale to provide when
+    # no enivornment variables are set, or are set to empty strings.  However,
```

Comment:
Typo in the comment: enivornment -> environment

Reply:
Heh, I'll just fix that and squash...